index.html

```diff
@@ -0,0 +1,251 @@
+                        <div class="f2-arrow"></div>
+                    </div>
+                </div>
+          
+                <div class="third-section">
+                    <div class="f3-left"></div>
+                    <div class="f3-right">
+                        <div class="f2-date"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-by"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f3-arrow"></div>
+                    </div>
+                    <div class="f3-right">
+                        <div class="f2-date"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-by"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f3-arrow"></div>
+                    </div>
+                </div>
+           
+                <div class="third-section">
+                    <div class="f3-right">
+                        <div class="f2-date"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-by"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f3-arrow"></div>
+                    </div>
+                    <div class="f3-right">
+                        <div class="f2-date"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-title"></div>
+                            <div class="f2-by"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
+                            <div class="f2-text"></div>
```

Comment:
## Each article
 
It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))


- [ ] Instead of `<div>` tag use `<article>` to each f2 class on the entire HTML file